src/Drupal/Commands/core/DeployHookCommands.php

```diff
@@ -153,4 +153,22 @@ public function run()
         $this->logger()->log($level, dt('Finished performing deploy hooks.'));
         return $success ? self::EXIT_SUCCESS : self::EXIT_FAILURE;
     }
+
+    /**
+     * Mark all deploy hooks as having run.
+     *
+     * @usage deploy:hook-skip
+     *   Skip all pending deploy hooks.
+     *
+     * @command deploy:hook-skip
```

Comment:
How about the name `deploy:mark-complete`? Change `@usage` description and method names to match.

Reply:
great suggestion. naming is hard